File: java/beginner/006_Arrays.java
// [6] 배열 기초 - Arrays
// 레벨: 2 | Java의 배열을 다루는 방법을 배웁니다

import java.util.Arrays;

public class ArraysDemo {

    public static void main(String[] args) {

        int[] scores = {85, 92, 78};
        System.out.println(scores[0]);
        System.out.println(scores.length);

        for (int s : scores) {
            System.out.println(s);
        }

        Arrays.sort(scores);
        System.out.println(Arrays.toString(scores));

        int idx = Arrays.binarySearch(scores, 92);
        System.out.println(idx);

        int[] copy = Arrays.copyOf(scores, scores.length);
        System.out.println(Arrays.toString(copy));
    }
}

File: csharp/beginner/006_Arrays.cs
// [6] 배열 기초 - Arrays
// 레벨: 2 | C#의 배열을 다루는 방법을 배웁니다

using System;

class Program
{
    static void Main()
    {
        int[] a = { 3, 1, 2 };
        Console.WriteLine(a[0]);
        Console.WriteLine(a.Length);

        foreach (var x in a)
            Console.Write(x);

        Console.WriteLine();

        Array.Sort(a);
        Console.WriteLine(a[0]);
    }
}
